examples/knative/powershell/kn-ps-slack/README.md

```diff
@@ -57,24 +57,33 @@ RelationLink      : {}
 # Step 3 - Deploy
 
 > **Note:** The following steps assume a working Knative environment using the
-`default` Rabbit broker. The Knative `service` and `trigger` will be installed in the
-`vmware-functions` Kubernetes namespace, assuming that the broker is also available there.
+`default` Rabbit `broker`. The Knative `service` and `trigger` will be installed in the
+`vmware-functions` Kubernetes namespace, assuming that the `broker` is also available there.
 
-Update the `secret` file with your Slack webhook URL and then create the kubernetes secret which can then accessed from within the function by using the environmental variable named called `SLACK_SECRET`.
+Push your container image to an accessible registry such as Docker once you're done developing and testing your function logic.
 
-```bash
+```console
+docker push <docker-username>/kn-pcli-tag:1.0
+```
+
+Update the `tag_secret.json` file with your Slack webhook configurations and then create the kubernetes secret which can then be accessed from within the function by using the environment variable named called `SLACK_SECRET`.
+
+```console
 # create secret
 
-kubectl -n vmware-functions create secret generic slack-secret --from-file=SLACK_SECRET=secret
+kubectl -n vmware-functions create secret generic slack-secret --from-file=SLACK_SECRET=slack_secret.json
+
+# update label for secret to show up in VEBA UI
+kubectl -n vmware-functions label secret slack-secret app=veba-ui
 ```
 
-Edit the `function.yaml` file with the name of the container image from Step 1 if you made any changes. If not, the default VMware container image will suffice. By default, the function deployment will filter on the `VmPoweredOffEvent` vCenter Server Event. If you wish to change this, update the `subject` field within `function.yaml` to the desired event type.
+Edit the `function.yaml` file with the name of the container image from Step 1 if you made any changes. If not, the default VMware container image will suffice. By default, the function deployment will filter on the `DrsVmPoweredOnEvent` vCenter Server Event. If you wish to change this, update the `subject` field within `function.yaml` to the desired event type.
```

Comment:
Since this example is for powered off event, shouldn't this be the default trigger used in this example then?